src/api-service/__app__/heartbeat/__init__.py

```diff
@@ -5,20 +5,44 @@
 
 import json
 import logging
+from typing import Dict
 
 import azure.functions as func
-from onefuzztypes.models import HeartbeatEntry
+from onefuzztypes.models import NodeHeartbeatEntry, TaskHeartbeatEntry
+from pydantic import ValidationError
 
 from ..onefuzzlib.dashboard import get_event
-from ..onefuzzlib.heartbeat import Heartbeat
+from ..onefuzzlib.heartbeat import NodeHeartbeat, TaskHeartbeat
+
+
+def try_add_task_heartbeat(raw: Dict) -> bool:
```

Comment:
How about making these methods named `try_add()` on `TaskHeartbeat` and `NodeHeartbeat`, respectively?